src/docs/src/documentation/content/xdocs/zookeeperProgrammers.xml

```diff
@@ -841,9 +841,9 @@
     itself, ZooKeeper associates all the ids that correspond to a
     client with the clients connection. These ids are checked against
     the ACLs of znodes when a clients tries to access a node. ACLs are
-    made up of pairs of <emphasis>(scheme:expression,
+    made up of pairs of <emphasis>(scheme:id,
```

Comment:
I'm not sure if this is the best way to clarify here. As demonstrated below with the ip address example, the second field can be an "expression" that matches against ids. Although in the code we occasionally refer to the second term as an "id" (`ap.matches(authId.getId(), id.getId())` in `PrepRequestProcessor`) we do also refer to it as an "expression" in other places (`boolean matches(String id, String aclExpr)` in `AuthenticationProvider`).

I think continuing to refer to the second term as an "expression" and explaining exactly what an "expression" is may be clearer.